be/src/io/fs/stream_load_pipe_reader.h

```diff
@@ -0,0 +1,97 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#pragma once
+
+#include <condition_variable>
+#include <deque>
+
+#include "io/fs/file_reader.h"
+#include "runtime/message_body_sink.h"
+
+namespace doris {
+namespace io {
+
+const size_t kMaxPipeBufferedBytes = 4 * 1024 * 1024;
+
+class StreamLoadPipeReader : public MessageBodySink, public FileReader {
```

Comment:
```suggestion
class StreamLoadPipe : public MessageBodySink, public FileReader {
```

Because it is not just a "Reader"